fdbclient/NativeAPI.actor.cpp

```diff
@@ -221,6 +221,37 @@ void DatabaseContext::addSSIdTagMapping(const UID& uid, const Tag& tag) {
 	ssidTagMapping[uid] = tag;
 }
 
+void DatabaseContext::getLatestCommitVersion(const StorageServerInterface& ssi,
+                                             Version readVersion,
+                                             VersionVector& latestCommitVersion) {
+	latestCommitVersion.clear();
+
+	if (ssVersionVectorCache.getMaxVersion() == invalidVersion) {
+		return;
+	}
+
+	// Error checking (based on the assumption that the read version was not obtained
+	// from the client's grv cache).
+	if (readVersion > ssVersionVectorCache.getMaxVersion()) {
+		TraceEvent(SevError, "GetLatestCommitVersion")
```

Comment:
can this error message in the log be more descriptive?

Reply:
How about "ReadVersionExceedsVersionVectorMaxVersion"?

Reply:
or just ReadVersionExceedsVersionVectorMax